features/configuration_files/reject_setting.feature

```diff
@@ -16,43 +16,8 @@ Feature: `reject` configuration setting
           - solid_argument
           - nifty_argument
       UncommunicativeModuleName:
-        reject: Dummy
-      """
-    And a file named "smelly.rb" with:
-      """
-      # Should report UncommunicativeModuleName
-      class Dummy
-        # Should report UncommunicativeMethodName and UncommunicativeParameterName
-        def awesome_helper(solid_argument); solid_argument; end
-        # Should report UncommunicativeMethodName and UncommunicativeParameterName
-        def little_helper(nifty_argument); nifty_argument; end
-        # Should not report UncommunicativeMethodName and UncommunicativeParameterName
-        def meth(argument); argument; end
-      end
-      """
-    When I run `reek -c config.reek smelly.rb`
-    Then it reports:
-    """
-    smelly.rb -- 5 warnings:
-      [4]:UncommunicativeMethodName: Dummy#awesome_helper has the name 'awesome_helper' [https://github.com/troessner/reek/blob/master/docs/Uncommunicative-Method-Name.md]
-      [6]:UncommunicativeMethodName: Dummy#little_helper has the name 'little_helper' [https://github.com/troessner/reek/blob/master/docs/Uncommunicative-Method-Name.md]
-      [2]:UncommunicativeModuleName: Dummy has the name 'Dummy' [https://github.com/troessner/reek/blob/master/docs/Uncommunicative-Module-Name.md]
-      [4]:UncommunicativeParameterName: Dummy#awesome_helper has the parameter name 'solid_argument' [https://github.com/troessner/reek/blob/master/docs/Uncommunicative-Parameter-Name.md]
-      [6]:UncommunicativeParameterName: Dummy#little_helper has the parameter name 'nifty_argument' [https://github.com/troessner/reek/blob/master/docs/Uncommunicative-Parameter-Name.md]
-    """
-
-  Scenario: reject regexes as list and as single item
```

Comment:
The git diff is very unfortunate here - very hard to see what's going on here. Basically we had the same scenario twice, only the second time with "reject regexes as list and as single item". I just removed the now obsolete 2n scenario.